python/paddle/fluid/tests/unittests/mkldnn/test_sum_mkldnn_op.py

```diff
@@ -86,4 +86,6 @@ def test_check_grad(self):
 
 
 if __name__ == '__main__':
+    from paddle import enable_static
```

Comment:
Just a comment, not asking for changes. Baidu recently added STATIC_MODE_TESTING_LIST, python unit test do not require enable_static() any more.

Reply:
ok, will try to remove it in next PR